git-extra/notepad

```diff
@@ -15,7 +15,12 @@ die "Usage: $0 <file>"
 
 if test -f "$1"
 then
-	unix2dos.exe "$1"
+	commitencoding="$(git config i18n.commitencoding)" ||
+		commitencoding=""
```

Comment:
But `git config` doesn't write anything to `stdout` in that case, so `"$commitencoding"` should still be an empty string.